examples/2- My first transaction.ipynb

```diff
@@ -0,0 +1,346 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import nibiru\n",
+    "import nibiru.common # Common values and types to run\n",
+    "import nibiru.msg # Messages package for transactions\n",
+    "import json"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nibiru.__version__ # 0.3.2 is compatible with nibiru chain v0.14"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "host = \"localhost\"\n",
+    "\n",
+    "network = nibiru.Network(\n",
+    "    lcd_endpoint=f'http://{host}:1317',\n",
+    "    grpc_endpoint=f'{host}:9090',\n",
+    "    chain_id='nibiru-localnet-0',\n",
+    "    fee_denom='unibi',\n",
+    "    env='local',\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Transaction types"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are 3 ways to post transactions with the python Sdk, having to do with the synchronization needed by the user.\n",
+    "\n",
+    "```python\n",
+    "class TxType(Enum):\n",
+    "    \"\"\"\n",
+    "    The TxType allows you to chose what type of synchronization you want to use to send transaction\n",
+    "    \"\"\"\n",
+    "\n",
+    "    SYNC = 1\n",
+    "    \"\"\"\n",
+    "    The CLI waits for a CheckTx execution response only.\n",
+    "    Each full-node that receives a transaction sends a CheckTx to the application layer to check for validity, and\n",
+    "    receives an abci.ResponseCheckTx. If the Tx passes the checks, it is held in the nodes' Mempool , an in-memory pool\n",
+    "    of transactions unique to each node pending inclusion in a block - honest nodes will discard Tx if it is found to\n",
+    "    be invalid.\n",
+    "\n",
+    "    Prior to consensus, nodes continuously check incoming transactions and gossip them to their peers.\n",
+    "    \"\"\"\n",
+    "\n",
+    "    ASYNC = 2\n",
+    "    \"\"\"\n",
+    "    The CLI returns immediately (transaction might fail silently).\n",
+    "    If you send a transaction with this option, it is recommended to query the transaction output using the hash of the\n",
+    "    transaction given by the output of the tx call.\n",
+    "    \"\"\"\n",
+    "\n",
+    "    BLOCK = 3\n",
+    "    \"\"\"\n",
+    "    The tx function will wait unitl the tx is be committed in a block.\n",
+    "    This have the effect of having the full log of the transaction available with the output of the method. These logs\n",
+    "    will include information as to coin sent and received, states changed etc.\n",
+    "    \"\"\"\n",
+    "```"
```

Comment:
This part's formatting looks wrong
![image](https://user-images.githubusercontent.com/1250307/188471165-457ac728-8025-41c5-adfa-32dfb89de6df.png)